share/tools/create_manpage_completions.py

```diff
@@ -975,6 +974,16 @@ def get_paths_from_man_locations():
     # If we can't have the OS interpret $MANPATH, just use it as-is (gulp).
     if not parent_paths and os.getenv("MANPATH"):
         parent_paths = os.getenv("MANPATH").strip().split(":")
+    # Fallback: With mandoc (OpenBSD, embedded Linux) and NetBSD man, the only way to get the default manpath is by reading /etc.
```

Comment:
No, NetBSD man has `man -p`, as the comment stated.

Reply:
NetBSD's `man -p` actually prints the section directories instead of the manpath directories -- that was my comment that I added, and it was in slight error. We could special-case `man -p` to take the `dirname` of each entry -- or we could parse the `man.conf` file as in https://netbsd.gw.com/cgi-bin/man-cgi?man.conf+5+NetBSD-current